amundsen_application/static/js/components/common/Table/index.tsx

```diff
@@ -1,54 +1,123 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
 import * as React from 'react';
 
+import ShimmeringResourceLoader from '../ShimmeringResourceLoader';
+
 import './styles.scss';
 
 export interface TableColumn {
   title: string;
   field: string;
-  horAlign?: 'left' | 'right' | 'center';
-  // width?: number;
   // className?: string;
+  // horAlign?: 'left' | 'right' | 'center';
```

Comment:
What are these comments for?

Reply:
These are proposals for the API of the table, happy to discuss the naming and if they are needed!